crates/humanode-runtime/src/lib.rs

```diff
@@ -711,6 +713,7 @@ impl pallet_ethereum::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
     type PostLogContent = PostBlockAndTxnHashes;
+    type ExtraDataLength = ConstU32<30>;
```

Comment:
What's this? I'll need to read up on frontier, as I haven't seen this one

Reply:
I see! Why `30` though, and not something round like `32`? Also, why not more, like `256`? Interesting...

Reply:
The value has been taken from https://github.com/moonbeam-foundation/moonbeam/blob/00d16ad12a31d2ecab50646d914d57ebb83681e0/runtime/moonbeam/src/lib.rs#L630.

I guess, here is the answer - https://github.com/humanode-network/frontier/blob/a487de275b0bfa60f924fef5b3d6ba446fad1edd/frame/ethereum/src/lib.rs#L572

<img width="581" alt="image" src="https://github.com/humanode-network/humanode/assets/39522748/56a33302-7bfd-4210-8605-bdf472c05d01">

Reply:
My bad, misreading some code on my end. So, it's an open question for now. 

As I sad, 30 has been taken from moonbeam runtime.

Reply:
Got it, will do.

Reply:
https://github.com/humanode-network/humanode/pull/829/commits/c887894d81e722daa6a4030a38be846994e895e5

Reply:
The issue has been created - https://github.com/humanode-network/humanode/issues/846.